doc/rst/platforms/cray.rst

```diff
@@ -535,9 +510,7 @@ When hugepages are used with the ugni comm layer, tasking layers
 cannot use guard pages for stack overflow detection.  Qthreads tasking
 can only use guard pages for stack overflow detection, so if ugni
 communications is combined with qthreads tasking, overflow detection is
-turned off completely.  Muxed tasking can use guard pages for stack
-overflow detection, but it can also drop back to synchronous overflow
-detection, as described below, with ``CHPL_COMM=ugni`` and hugepages.
+turned off completely.
```

Comment:
wait actually, I don't think this is wrong. It reads:

"""
When hugepages are used with the ugni comm layer, tasking layers cannot use guard pages for stack overflow detection. Qthreads tasking can only use guard pages for stack overflow detection, so if ugni communications is combined with qthreads tasking, overflow detection is turned off completely.
"""

That section starts with "When hugepages are used with the ugni comm layer..." so I think this is only talking about what the behavior is when a hugepage module is loaded. 

I don't disagree that the documentation could be improved, but I don't think it's wrong, is it?